app/app.go

```diff
@@ -68,6 +81,10 @@ func buildScaledContext(ctx context.Context, kubeConfig rest.Config, cfg *Config
 }
 
 func Run(ctx context.Context, kubeConfig rest.Config, cfg *Config) error {
+	getID := func() string {
```

Comment:
whats the point of this wrapper function? only place it is called is on line 107 and getid(cfg) could have just been called directly